cli/collection_create.go

```diff
@@ -17,14 +17,25 @@ import (
 	"github.com/spf13/cobra"
 
 	"github.com/sourcenetwork/defradb/client"
+	"github.com/sourcenetwork/defradb/internal/db"
 )
 
 func MakeCollectionCreateCommand() *cobra.Command {
 	var file string
+	var encryptionKey string
 	var cmd = &cobra.Command{
-		Use:   "create [-i --identity] <document>",
+		Use:   "create [-i --identity] [-e --encryption] <document>",
```

Comment:
todo: The encryption key shouldn't be passed on doc creation. In a local first context, we can tell Defra to encrypt the document. The `-e`  flag can be that signal. Defra will then create the encryption key, store it like you're doing now and encrypt the fields. In a future multi-user context, we wouldn't want to trust the node to use the encryption key that we pass to it so the document would have to be encrypted before prior to sending it.

Reply:
made defra generate keys